src/status_im2/subs/contact.cljs

```diff
@@ -84,6 +84,26 @@
        sort
        vals)))
 
+(defn distinct-by [f coll]
```

Comment:
To me, this kind of function should be moved to `utils/core.cljc` and it should be unit tested.